Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ruby:3.2.2
+RUN mkdir /appbuild
+COPY . /appbuild
+WORKDIR /appbuild/docs
+RUN ls -la
+RUN bundle install
+RUN which bundle
+CMD ["bundle", "exec", "jekyll", "serve", "-l"]
```

Comment:
Unfortunately we have to build a jekyll image from a ruby base instead of just running our config because the `jekyll/jekyll` image seems persistently broken with permissions issues.